RELEASE.txt

```diff
@@ -18,64 +18,82 @@ make a PR to update these notes after you are done with the release. ;-)
+  7. Copy relevant deprecations from ``release_<major>_<minor>.txt``
      to ``release_dev.txt``.
 
-- Update the version number in ``skimage/__init__.py`` and commit.
+- Submit the release notes for review by other project maintainers:
+
+  - Create a temporary PR from v<major>.<minor>.x branch to master
+    (at this point, the difference should show the full contents of the
+     release notes).
+  - Discuss, and make the changes directly to v<major>.<minor>.x branch.
+  - Once the consensus is found, close the PR and go to the next step.
+
+- Update the version number to stable in ``skimage/__init__.py`` and commit.
 
 - Update the docs:
 
   - Edit ``doc/source/_static/docversions.js`` and commit
   - Build a clean version of the docs.  Run ``python setup.py install`` in the
-    root dir, then ``rm -rf build; make html`` in the docs.
+    root dir, then ``rm -rf build; make html`` in the ``doc`` dir.
   - Build using ``make gh-pages``.
   - Update the symlink to ``stable``.
   - Push upstream: ``git push origin gh-pages`` in ``doc/gh-pages``.
 
 - Add the version number as a tag in git::
 
-   git tag -s [-u <key-id>] v<major>.<minor>.0
+   git tag -s -m <github_release_message> [-u <key-id>] v<major>.<minor>.0
 
-  (If you do not have a gpg key, use -m instead; it is important for
-  Debian packaging that the tags are annotated)
+  (If you do not have a GPG key, follow the tutorial to set it up:
+   https://help.github.com/articles/signing-commits-with-gpg/)
 
-- Push the new meta-data to github::
+- Push the new tag to GitHub::
 
-   git push --tags upstream master
+   git push upstream v<major>.<minor>.0
 
   (where ``upstream`` is the name of the
    ``github.com:scikit-image/scikit-image`` repository.)
 
+- Make sure you have all the required write permissions (if not, you will need
+  to ask Stéfan or Juan to grant you the access), specifically, to:
+
+  - https://github.com/scikit-image/scikit-image-wheels
+  - https://pypi.org/project/scikit-image/
+  - https://github.com/scikit-image/scikit-image-web
+
 - Build the package wheels (pre-compiled binaries) for various platforms:
 
   - Clone https://github.com/scikit-image/scikit-image-wheels.
   - Update its ``.travis.yml`` file so that ``BUILD_COMMIT`` points to this
-    release tag (e.g. ``v0.13.0``).
+    release tag (e.g. ``v0.14.0``).
   - Commit and push.
   - Wait until the corresponding wheels appear at
     http://wheels.scikit-image.org (about 15 minutes).
 
-- Download the wheels and upload them to PyPI (you may need to ask Stéfan or
-  Juan to give you push access to the scikit-image package on PyPI).
+- Upload the wheels to PyPI:
 
   - Make sure ``twine`` is available. You can install it with
     ``pip install twine``.
   - Download ``wheel-uploader`` [1]_ and place it on your PATH.
   - Run ``tools/upload_wheels.sh``.
 
-.. [1] https://github.com/MacPython/terryfy/master/wheel-uploader
+.. [1] https://github.com/MacPython/terryfy/blob/master/wheel-uploader
```

Comment:
Does this take care of Windows as well?

Reply:
Erm... no :). This time @matthew-brett uploaded the Windows wheels. Not sure if there is an automated way of doing this in our infrastructure yet.